pandas/tests/test_base.py

```diff
@@ -1219,10 +1224,19 @@ def test_iter_box(self):
     (np.array([0, 1], dtype=np.int64), np.ndarray, 'int64'),
     (np.array(['a', 'b']), np.ndarray, 'object'),
     (pd.Categorical(['a', 'b']), pd.Categorical, 'category'),
-    (pd.DatetimeIndex(['2017', '2018']), np.ndarray, 'datetime64[ns]'),
-    (pd.DatetimeIndex(['2017', '2018'], tz="US/Central"), pd.DatetimeIndex,
+    # Ughh this is a mess. We want to keep Series._values as
```

Comment:
probably discussed elsewhere, why do we want Series._values to stay an ndarray?

Reply:
`Series._values` is eventually Block.internal_values, which for tz-naive is a DatetimeBlock, which holds ndarrays.

I'm not sure what special casing tz-naive to return a DatetimeArray would break.